server.js

```diff
@@ -2,7 +2,7 @@ const express = require('express')
 const app = express()
 
 app.get('/ping',(req,res)=>{
-    res.send('Ping')
+    res.json({message:'Pong'})
 })
 
 app.listen(3000,()=>{
```

Comment:
Consider using a constant for the port number instead of hardcoding it. This will make it easier to change the port in the future if needed. [medium]